wiki/Ranking_Criteria/osu!catch/en.md

```diff
@@ -51,10 +63,10 @@ Overall rules and guidelines apply to every kind of osu!catch difficulty. Rhythm
 - **Dashes and hyperdashes should not be used when the destination of the dash or hyperdash is located near the left or right border of the playfield.** This creates an uncomfortable movement as the catcher is forcibly stopped upon reaching the border of the playfield (x:16 on the left and x:496 on the right).
 - **Ensure combos do not reach unreasonable lengths.** Caught fruits will stack up on the plate and can potentially obstruct the player's view. Bear in mind that slider tails, repeats and spinner bananas also count as "fruits". New Combos should be placed regularly to clear the plate and avoid this.
 - **Try to have at least one spinner in each difficulty to create variety in the map and fluctuation among scores.** However, if a spinner just doesn't fit anywhere in the song, then there's no need to force one.
-- **Overall Difficulty should have the same value as the Approach Rate.** This is just a standardized value, as Overall Difficulty does not affect gameplay nor the amount of fruits a spinner has, just the maximum score of a difficulty. However, if a difficulty uses a lower Approach Rate than one or more difficulties in the difficulty level below it, then the Overall Difficulty should be equal to that of the highest Overall Difficulty value in the previous difficulty level.
+- **Overall Difficulty should have the same value as the Approach Rate.** This is just a standardized value, as Overall Difficulty does not affect gameplay nor the number of fruits a spinner has, just the maximum score of a difficulty. However, if a difficulty uses a lower Approach Rate than one or more difficulties in the difficulty level below it, then the Overall Difficulty should be equal to that of the highest Overall Difficulty value in the previous difficulty level.
```

Comment:
```suggestion
- **Overall Difficulty should have the same value as the Approach Rate.** This is just a standardised value, as Overall Difficulty does not affect gameplay nor does the number of fruits a spinner has; only the maximum score of a difficulty matters. However, if a difficulty uses a lower Approach Rate than one or more difficulties in the difficulty level below it, then the Overall Difficulty should be equal to that of the highest Overall Difficulty value in the previous difficulty level.
```
the wording confuses me on what you're trying to communicate there (second sentence). If my attempt is wrong please do correct

Reply:
It is indeed a bit unclear, although your changes would make the last part of the sentence incorrect. I'll rewrite the sentence once all the other suggestions are applied